toolkits/landcover/test/int/dataset.int.test.js

```diff
@@ -84,4 +84,11 @@ withEarthEngine('Dataset', function() {
           done();
         });
   });
+
+  it('addTasseledCap()', function(done) {
+    expect(function() {
+      var l8 = TestDataset(TEST_VALUES).addTasseledCap();
```

Comment:
You can remove the `var l8 =` since it's never used here.

Reply:
Done